cmd/ddev/cmd/exec.go

```diff
@@ -67,9 +67,14 @@ ddev exec --raw -- ls -lR`,
 		}
 
 		_, _, err = app.Exec(opts)
+		quiet, _ := cmd.Flags().GetBool("quiet")
 
 		if err != nil {
-			util.Failed("Failed to execute command `%s`: %v", opts.Cmd, err)
+			if !quiet {
+				util.Failed("Failed to execute command `%s`: %v", opts.Cmd, err)
+			} else {
+				os.Exit(1)
```

Comment:
Don't we know what error it exited with, and can use that here? The reality is we should probably be exiting with a useful error code above instead of util.Failed().

Reply:
And do we want to exit with an error, or ignore the error completely (os.Exit(0))?

Reply:
I would go with an error code so that the caller could still check with `$?` that there was an error (even when the DDEV error message is supressed). 😊

If it is easily possible, the error code of the `ls` command would be a nice-to-have feature I think so that you can respond to the ls (or any other command) the right way? 🤔